bundle/edu.gemini.model.p1.submit/src/main/scala/edu/gemini/model/p1/submit/FutureSubmission.scala

```diff
@@ -67,21 +67,7 @@ case class FutureSubmission(dest: SubmitDestination, url: String, proposal: Prop
         Option(http.getErrorStream).map(s => parse(read(s))).collect(errorAdapter).getOrElse {
           ServiceError(dest.some, proposal.proposalClass.some, 0, "An unexpected error happened in the submission server.")
         }
-    }
   }
-
-}
-
-object FutureSubmission {
-
-  val sslFactory = {
-    // Load keystore from the bundle
-    val in = this.getClass.getResourceAsStream("/phase1.jks")
```

Comment:
Do you know the history of this?  Just wondering why we did it this way before.

Reply:
I did it in 2015 when we switched to SSL. I have a vague recollection that it was using an external certificate. Let's encrypt was probably already in use then before being used for other gemini servers